native/core/src/execution/datafusion/planner.rs

```diff
@@ -1599,6 +1599,18 @@ impl PhysicalPlanner {
         }
     }
 
+    /// Find DataFusion's built-in window function by name.
+    fn find_df_window_function(name: &str) -> Option<WindowFunctionDefinition> {
+        if let Some(f) = find_df_window_func(name) {
+            Some(f)
+        } else {
+            match name {
+                "count" => Some(WindowFunctionDefinition::AggregateUDF(count_udaf())),
+                _ => None,
```

Comment:
Can we look for aggregate udf from FunctionRegistry? It is similar to how we look for scalar function from `FunctionRegistry`.

Reply:
You can check `create_comet_physical_fun`.

Reply:
Changed. Thanks